src/core/datatypes/phylogenetics/character/CharacterTranslator.cpp

```diff
@@ -75,6 +76,21 @@ RevBayesCore::AbstractDiscreteTaxonData* CharacterTranslator::translateCharacter
             trans_taxon_data = translateToCodon( d );
         }
     }
+    else if ( type == "Doublet")
+    {
+        if ( d.getCharacter(0).getDataType() == "DNA" )
+        {
+            trans_taxon_data = translateToDoubletFromDna( dynamic_cast< const DiscreteTaxonData<DnaState>& >(d) );
+        }
+        else if ( d.getCharacter(0).getDataType() == "RNA" )
+        {
+            trans_taxon_data = translateToDoubletFromRna( dynamic_cast< const DiscreteTaxonData<RnaState>& >(d) );
+        }
+        else
+        {
+            trans_taxon_data = translateToCodon( d );
```

Comment:
@bredelings Should this be `trans_taxon_data = translateToDoublet( d );`?

Reply:
Yes!  Thanks for catching that.   That routine throws an exception, so we would have gotten the wrong error message.